modules/vcf2maf/1.3/vcf2maf.smk

```diff
@@ -15,31 +15,55 @@ import sys, os
 from os.path import join
 
 import oncopipe as op
+import pandas as pd
 
 # Setup module and store module-specific configuration in `CONFIG`
 CFG = op.setup_module(
     name = "vcf2maf",
     version = "1.3",
-    subdirectories = ["inputs","decompressed","vcf2maf","crossmap","outputs"]
+    subdirectories = ["inputs","decompressed","vcf2maf","crossmap", "normalize", "outputs"]
 )
 
 # Define rules to be run locally when using a compute cluster
 localrules:
     _vcf2maf_input_vcf,
     _vcf2maf_gnomad_filter_maf,
-    _vcf2maf_output_maf,
+    _vcf2maf_output_maf_native,
+    _vcf2maf_output_maf_projection,
     _vcf2maf_crossmap,
     _vcf2maf_all
 
-VCF2MAF_GENOME_VERSION_MAP = {
-    "grch37": "GRCh37",
-    "hg38": "GRCh38",
-    "hs37d5": "GRCh37"
-}
-
 #set variable for prepending to PATH based on config
 VCF2MAF_SCRIPT_PATH = CFG['inputs']['src_dir']
 
+# This is going to be annoying, but to normalize prefixes, we need to determine
+# 1) Which iteration of the genome build something is (i.e. GRch37 or GRCh38)
+# 2) If a genome build is chr-prefixed or not
+# To determine this, lets load the reference config and re-parse it
+VCF2MAF_REFERENCE_CONFIG = CFG["options"]["reference_config"]  # TEMP, remove later
```

Comment:
what do you mean by TEMP here? Where is it getting removed?

Reply:
Sorry I forgot to delete this comment (originally this was a hard-coded file path for testing purposes)